www/docs/en/dev/config_ref/index.md

```diff
@@ -167,6 +167,17 @@ all of them:
 
         <preference name="BackgroundColor" value="0xff0000ff"/>
 
+  Applies to Windows.
+  
+- CSS Color names are also supported:
+
+```
+<platform name="windows">
+    <preference name="BackgroundColor" value="turquoise"/>
+</platform>
+```
+  __NOTE__: `transparent` value will cause black/white background color in case of Dark/Light theme accordingly.
+
   Applies to Android and BlackBerry. Overrides CSS otherwise available
```

Comment:
@nikhilkh, Actually I believe the phrase `Applies to Android and BlackBerry. Overrides CSS otherwise available across _all_ platforms, for example: `body{background-color:blue}`.` says about page style taken from `BackgroundColor` preference, which is not supported on Windows.
I will update `Applies to Windows.` to `Windows Quirk` to avoid confusing with the rest of the doc though.